pixi.toml

```
@@ -32,8 +32,14 @@ mypy = ">=1.11,<1.12"
 psutil = ">=6.0.0,<7"
 # For running tests in parallel, use this instead of regular pytest
 pytest-xdist = ">=3.6.1,<4"
+rich = ">=13.7.1,<14"
+toml = ">=0.10.2,<0.11"
 
 [feature.pytest.tasks]
+test-common-wheels = { cmd = "pytest -n logical tests/wheel_tests/", depends-on = [
+  "build",
+] }
+test-common-wheels-ci = { cmd = "pytest -n logical tests/wheel_tests/" }
```

```suggestion
test-common-wheels-ci = { cmd = "pytest -n logical tests/wheel_tests/" }
test-common-wheels-dev = { cmd = "pytest -n logical tests/wheel_tests/", depends-on = [
  "build",
] }
```

In order to stay consistent with `test-integration`